### DISCUSSION TITLE
I do not see the auto launch mode in inav configurator

### DISCUSSION THREAD

**Discussion by IvanBiv** [iNavFlight/inav-configurator#1403] [Q&A] [answered]
Hello.
I do not see the auto launch mode in inav configurator.
Inav configurator 3.0.2.
Should auto launch mode be in inav configurator?

**Comment by avsaase** [comment, marked as answer]
Do you have it permanently enabled on the configuration tab?

**Reply by IvanBiv** [reply]
No. This item does not exist at all.  The toggle switch "Hide unused modes" is off.

**Reply by avsaase** [reply]
I think it is called NAV LAUNCH in the modes tab.

**Reply by IvanBiv** [reply]
I think it is my error. But where is it? I don't understand where item NAV LAUNCH.
![IMG_20211119_121712_588](https://user-images.githubusercontent.com/7404103/142597674-4208a3ef-46d0-4a21-a0f4-b31be1998759.jpg)
![IMG_20211119_121710_912](https://user-images.githubusercontent.com/7404103/142597685-6b2b03a3-186f-4615-a834-01de2fa223fd.jpg)
I th
![IMG_20211119_121658_248](https://user-images.githubusercontent.com/7404103/142597729-d1dbc14a-cd5c-4b05-a698-56e1ff3d3d97.jpg)
![IMG_20211119_121701_454](https://user-images.githubusercontent.com/7404103/142597736-e6356858-7011-4030-8b0b-1901bd93d32d.jpg)
![IMG_20211119_121703_374](https://user-images.githubusercontent.com/7404103/142597746-9e1448f0-b880-44d8-bd6f-2486d3583eff.jpg)
![IMG_20211119_121705_244](https://user-images.githubusercontent.com/7404103/142597750-3132bec3-2338-471e-bdde-899573d389e9.jpg)
![IMG_20211119_121706_889](https://user-images.githubusercontent.com/7404103/142597717-b94adaea-e0d3-4662-af39-666460764028.jpg)
![IMG_20211119_121709_119](https://user-images.githubusercontent.com/7404103/142597708-4ad755c5-0c80-430e-8ef9-861fb6ab1fdc.jpg)

**Comment by DzikuVx** [comment]
Fixed wing or Multirotor?

**Reply by IvanBiv** [reply]
Fixed wing

**Reply by IvanBiv** [reply]
?

**Comment by MrD-RC** [comment]
Can you show the bottom right switch block on the configuration page

**Reply by IvanBiv** [reply]
![inav_configuration_1](https://user-images.githubusercontent.com/7404103/142612965-f000f808-2a7d-44f3-bea2-e508b3c120f4.PNG)
![inav_configuration_2](https://user-images.githubusercontent.com/7404103/142612962-c941d446-8ad9-494e-aae1-7aa66f446ef9.PNG)

**Reply by avsaase** [reply]
![image](https://user-images.githubusercontent.com/880421/142616780-a5ebc643-0cb1-4687-9c43-8f683d05f5d3.png)

**Reply by IvanBiv** [reply]
It's reason! Thanks!
